cl/users/tests.py

```diff
@@ -329,3 +334,498 @@ def test_set_password_using_the_HTML(self) -> None:
             self.browser.current_url,
             f"{self.live_server_url}{reverse('password_reset_complete')}",
         )
+
+
+class SNSWebhookTest(TestCase):
+    def setUp(self) -> None:
+        test_dir = os.path.join(
```

Comment:
I'm constantly encouraging people to use Pathlib for things like this, but dont' feel obligated to upgrade this code. One day I'll get mean about it, I guess. :)

Reply:
Solved! I'm now using Pathlib :)